Makefile

```diff
@@ -46,4 +48,12 @@ release:
 	opam publish submit $(NAME_VERSION)
 	rm -rf $(NAME_VERSION)
 
+install:
+	ocamlfind remove ppx_deriving
+	grep -E '^[[:space:]]+' ppx_deriving.install | \
```

Comment:
I've looked after ocp-build, I have big hopes for it, but right now there is a clear lack of documentation for the tool which makes it a no-go for an use in a professional context.